src/commands/moderation/warn/warn.ts

```diff
@@ -100,7 +94,10 @@ const noModLogs = makeEmbed({
 
 export const warn: CommandDefinition = {
     name: 'warn',
-    requiredPermissions: ['BanMembers'],
+    requirements: {
+        permissions: ['BanMembers'],
```

Comment:
```suggestion
```
Not a comment on the PR, but @benw202 : I'd suggest removing the permissions, it really doesn't have much of a point with the roles there anyway.